docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Split boards with Dongles
+sidebar_label: Split & Dongle
+---
+
+ZMK supports split keyboards that communicate wirelessly using BLE. Commonly these keyboards are split into two halves, each with its own microcontroller and battery. 
+Usually the left half of the keyboard is given the split role of central, and the right half is given the split role of peripheral. 
+The central half is responsible for communicating with the peripheral half and the host device.
+
+Using this setup does have one drawback: the central half of the keyboard usually runs out of battery faster than the peripheral half. 
+This is because the central half is responsible for maintaining the connection with the peripheral half and the host device.
+
+To solve this problem, a third microcontroller can be used as the new central or as more widely known a dongle. 
+For example, if you are using nice!nano v2 as your keyboard halves, you can use any other microcontroller as the dongle. 
+Meaning that the dongle does not have to be the same microcontroller as the keyboard halves, but it can be.
+
+The dongle will act as a central device and communicate with the keyboard halves which will both act as peripherals.
+Dongles are usually connected to the host device using USB. This way the central half of the keyboard can be powered by the host device, 
+and the peripheral halves can be powered by their own batteries. This setup allows both halves of the keyboard to have a longer battery life.
+
+
+![Mermaid Connection](../assets/splits/mermaid-connection.png)
+
+{/* Here is an example diagram of a split keyboard with a dongle:
+
+```mermaid
+graph TD;
+    A[Computer/Host Device] <-- USB or BLE --> DONGLE[Dongle/Central];
+    DONGLE -. BLE .-> LEFT[Left Half/Peripheral];
+    DONGLE -. BLE .-> RIGHT[Right Half/Peripheral];
+```
+
+Dotted lines represent BLE connections.
+Solid lines represent USB connections. */}
+
+:::info
+For more information on how to set up a two split keyboard, refer to [New Keyboard Shield](../development/new-shield.mdx) under Guides.
+After setting up the split keyboard, you can follow the steps below to add a dongle to the setup.
+:::
+
+## Benefits/Disadvantages of using a Dongle
+
+Benefits:
+
+- Adding a dongle, enables both keyboard halves to act as peripherals, meaning equal battery life for both halves.
+- Easier to connect to the host device.
+
+Disadvantages:
+
+- More complex setup.
+- Extra microcontroller needed
+- Keyboard is unusable without the dongle.
+- Added latency due to the extra hop between the host device and the keyboard halves.
+
+
+
+## Defining a Dongle for a Keyboard 
+
+
+:::note
+In the examples below, we will be referencing the New Split Keyboard Shield guide files and will be building on top of that.
+:::
```

Comment:
As mentioned in our conversation, I'd like to see versions for unibodies documented as well. I'd also like to see what the setup looks like if you have a keyboard with an integrated MCU, i.e. a board but no shield.

Reply:
Added tabs for both split and unibody